azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/autoconfigure/ApplicationInsightsTelemetryAutoConfiguration.java

```diff
@@ -211,6 +211,10 @@ public InternalLogger internalLogger() {
         Map<String, String> loggerParameters = new HashMap<>();
         ApplicationInsightsProperties.Logger logger = applicationInsightsProperties.getLogger();
         loggerParameters.put("Level", logger.getLevel().name());
```

Comment:
Same here. The key literals should be using the constants defined in the class where they are used.

Reply:
same , should update and will do that.